clog/loggers.py

```diff
@@ -269,7 +269,7 @@ def close(self):
             self.stream_files[name].close()
 
     def _create_file(self, stream):
-        return open(os.path.join(config.log_dir, stream + '.log'), 'a', 0)
+        return open(os.path.join(config.log_dir, stream + '.log'), 'ab', 0)
```

Comment:
surely this'll cause a `TypeError` when attempting to write text (py3 `str`) into a binary log?